### PR TITLE
Automate prerelease package publishing

### PR DESCRIPTION
**Pull request by jkusa**
Needs https://github.com/yahoo/navi/pull/58

- Automate prerelease package publishing
- Add `travis_wait` to longer running steps

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,42 +1,40 @@
...
-  - npm install -g bower
-  - bower --version
-  - npm install phantomjs-prebuilt
-  - node_modules/phantomjs-prebuilt/bin/phantomjs --version
-
+- npm config set spin false
+- npm --version
+- npm install -g bower
+- bower --version
+- npm install phantomjs-prebuilt
```
Comment (balajimanoharan) [inline .travis.yml]: Unrelated to your PR, but anyways... since we have switched to chrome, do we still need these?

File: .travis.yml [hunk 2]
```diff
@@ -1,42 +1,40 @@
...
+- node_modules/phantomjs-prebuilt/bin/phantomjs --version
 install:
-  - npm install
-
+- travis_wait npm install
 script:
-  - lerna run test --scope $PACKAGE
+- travis_wait lerna run test --scope $PACKAGE
```
Comment (balajimanoharan) [inline .travis.yml]: Do we need the travis_wait?

File: scripts/ci-publish.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+###############################################################################
+# Config NPM
+###############################################################################
```
Comment (balajimanoharan) [inline scripts/ci-publish.sh]: Do we need these banners? I don't really like them here.

File: .travis.yml (hunk 1 above)
Comment (jkusa) [reply .travis.yml]: https://github.com/yahoo/navi/pull/58

File: .travis.yml (hunk 2 above)
Comment (jkusa) [reply .travis.yml]: We do, I was seeing build timeouts without it

File: scripts/ci-publish.sh (hunk 3 above)
Comment (jkusa) [reply scripts/ci-publish.sh]: The script was initially bigger and made more sense. Still, bash scripts can get unwieldy. I like setting a standard that can carry forward if/when this scripts evolves.

**Pull request merged**